pkg/machine/ignition/ignition.go

```diff
@@ -743,6 +776,37 @@ func (i *IgnitionBuilder) Build() error {
 	return i.dynamicIgnition.Write()
 }
 
+func GetNetRecoveryFile() string {
+	return `#!/bin/bash
+# Verify network health, and bounce the network device if host connectivity
+# is lost. This is a temporary workaround for a known rare qemu/virtio issue
+# that affects some systems
+
+sleep 120 # allow time for network setup on initial boot
```

Comment:
Is this sleep still needed with the systemd unit which has `recoveryUnit.Add("Unit", "After", "sshd.socket sshd.service")` ?

Reply:
sshd only has an after on network.target, which is quasi reliable:

 "network.target has very little meaning during start-up. It only indicates that the network management stack is up after it has been reached. Whether any network interfaces are already configured when it is reached is undefined  [snip]"

Since we only seem to see the problem with long running vms, my thinking was it was better to just wait a bit longer in the script  without disrupting  / delaying boot (e.g. using something like network-online.target).

Reply:
yeah, I agree sleeping for 2 minutes (or even 5 minutes or 1 hour or ... :) is no big deal given when the bug happens. I just wondered.